jenkins/vs/sonic-swss-build-pr/Jenkinsfile

```diff
@@ -10,10 +10,10 @@ pipeline {
                               userRemoteConfigs: [[url: 'https://github.com/Azure/sonic-swss',
                                                    refspec: '+refs/pull/*:refs/remotes/origin/pr/*']]])
                 }
-                copyArtifacts(projectName: 'sonic-sairedis-build', filter: '**/*.deb', target: 'sairedis', flatten: true)
+                copyArtifacts(projectName: 'vs/sonic-sairedis-build', filter: '**/*.deb', target: 'sairedis', flatten: true)
```

Comment:
@daall is this change related to issue ?

Reply:
No, I added this so I can test the change since the `jenkins_test` folder isn't in the `vs`.